docs/documentation/operate/operate-troubleshooting.md

```
@@ -31,6 +31,102 @@ Look for Prometheus query errors, auth failures, or TLS issues.
+| `kube_node_status_allocatable` | Node allocatable CPU, memory, and GPU capacity |
+| `kube_node_status_capacity` | Node CPU capacity (e.g. load ratio in node load monitoring) |
+| `kube_pod_container_status_last_terminated_reason` | OOM detection (`reason="OOMKilled"`) |
+| `kube_pod_container_status_restarts_total` | OOM / restart correlation |
+| `kube_node_spec_taint` | Exclude GPU nodes from certain cluster rollups (`key="nvidia.com/gpu"`) |
+| `kube_node_labels` | Exclude NVIDIA accelerator nodes from certain cluster rollups |
+
+#### node-exporter (node-level signals)
+
+| Metric | Used for |
+|--------|----------|
+| `node_cpu_seconds_total` | Cluster CPU utilization rollups (`mode=~"user|system"`) |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)